config/routes.rb

```diff
@@ -183,6 +183,7 @@
 
     resources :ownership_calls, only: :index
     resources :webauthn_credentials, only: :destroy
+    get 'webauthn_verification/:webauthn_token', to: 'webauthn_verifications#verify', as: 'webauthn_verification'
```

Comment:
This is fairly distant from the `resource :webauthn_verification` for the controller on L26. We should either move this up or move that one down.

Reply:
The one on L26 is for the `api/v1/` routes. This one is not an api route so I put it with the other webauthn related UI routes

Reply:
I didn't notice that distinction. Good call.